src/main/java/com/fasterxml/jackson/core/json/JsonWriteFeature.java

```diff
@@ -72,6 +72,15 @@
     @SuppressWarnings("deprecation")
     ESCAPE_NON_ASCII(false, JsonGenerator.Feature.ESCAPE_NON_ASCII),
 
+    /**
+     * Feature that determines whether to use standard Java code to write floats/doubles (default) or
+     * use the Schubfach algorithm which is faster. The latter approach may lead to small
+     * differences in the precision of the float/double that is written to the JSON output.
+     *
+     * @since 2.14
+     */
+    USE_FAST_DOUBLE_WRITER(false, JsonGenerator.Feature.USE_FAST_DOUBLE_WRITER),
```

Comment:
As with reader side, let's move this to more general `StreamWriteFeature` as it need not be JSON specific but applicable to most if not all textual format backends.